src/components/FolderCreationModal.jsx

```diff
@@ -185,6 +189,7 @@ const FolderCreationModal = ({
                   disabledStyle={elementStyles.disabled}
                   className={`${selectedFiles.size === 0 ? elementStyles.disabled : elementStyles.blue}`}
                   callback={saveHandler}
+                  isLoading={isLoading}
```

Comment:
If we're back to using `LoadingButton`, then we should remove the `isLoading` variable from `useMutation` and also remove it as a prop

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/369/commits/28a1bf897b7b5ec22117538ee4b4ed727466c9f3